openslides_backend/http/views/action_view.py

```diff
@@ -39,6 +39,7 @@ def action_route(self, request: Request) -> Tuple[ResponseBody, Optional[str]]:
 
         # Handle request.
         handler = ActionHandler(self.env, self.services, self.logging)
+        handler.set_translation_language(request.headers.get("Accept-Language"))
```

Comment:
and here you can set the language directly on the translate service

Reply:
Use the translator Singleton here.